testing.rst

```diff
@@ -211,6 +211,21 @@ Now you can use CSS selectors with the crawler. To assert that the phrase
         $crawler->filter('html:contains("Hello World")')->count()
     );
 
+.. versionadded:: 4.3
+
+    You can also use a shortcut::
+
+        $this->assertSelectorTextContains('html', 'Hello World');
+
+    This assertion will internally call ``$crawler->filter('html')``, which allows
+    you to use CSS selectors to filter any HTML element in the page and check for
+    its existence, attributes, text, etc.
```

Comment:
As the new method is much nicer than the previous one, let's update the code example directly using this assertion. We can then reduce this `.. versionadded::` directive to "The WebTestCase assertions were introduced in Symfony 4.3." and add the `.. seealso::` directive below it.

Reply:
I just added what you suggested, what do you think? 🙂